### DISCUSSION TITLE
No need for sermons

### DISCUSSION THREAD

**Discussion by netmender** [JenuelDev/Believers-Sword#38] [Ideas]
I like this bible software. I think it has a lot of potental. I would suggest that there should be no preinstalled sermon, Users should be allowed to embed or remove sermons. I think choosing sermons makes the software only for your personal use, if you really want to make it public for others then it is important you remove this sermons and allow people to make thier choices. 
Thank you

**Comment by JenuelDev** [comment]
Thanks for the suggestion.. I am currently making sermon page as a page to see other peoples sermons,,, currently there are just some dummy datas I added just for testing.. later on, I will be making the sermon page where you can choose sermons you like and maybe able to post your own sermons, wether by text or video

**Reply by netmender** [reply]
Thank you for the reply. Please how can I remove these sermons currently on sermon page because I don't need them. 
Thank you.

**Reply by JenuelDev** [reply]
I will be adding a button on the settings, maybe this afternoon :) This way you can either remove and not show sermon page page and tab :)

**Reply by JenuelDev** [reply]
Good day, I just released a new update for enabling and disabling some tabs. https://github.com/Bible-Projects/believers-sword-next/releases/tag/v0.2.7

**Reply by netmender** [reply]
Thank you

**Reply by JenuelDev** [reply]
If you have any suggestion please let us know :)

**Reply by netmender** [reply]
I think you should work on the clipnote. I made a note and afterward I could not delete it. Clipnotes on verse should be deletable if it is no longer needed. Secondly, It would better  if  notes are shown with an icon on the verse and when it is clicked, it  should pop up as a scrollable tooltips note.  Also, It would be better  to have possibilities to  select multiple verses and make one note for them.

**Reply by netmender** [reply]
I would like to ask what is your objective regarding this software.  Do you intend to make it a bible  software  for basic reading or have more features for study purpose? Do you intend to convert the software from open source to commercial later?  I have more suggestions but would like to know your intention.

**Reply by JenuelDev** [reply]
nope, I am sticking to making it free, and also making it a bible reading and basic studying as well.

**Reply by netmender** [reply]
Hello Bro Jenuel. Please, is it possible to have  the clipnote as a standalone software note?. If it is possible let me know so that we can discuss further.

**Reply by JenuelDev** [reply]
what do you man by a standalone?

**Reply by netmender** [reply]
I mean to have clipnote as a  sermon note software. I mean is it possible to seperate it from the bible software?

**Reply by JenuelDev** [reply]
oH I see, yeah,, so I am still working on note-taking,,:)

**Reply by netmender** [reply]
I would like to have a note taking software with my ministry name on it. I am planning to start a ministry for youth. I intend to give it free  for ministry purpose. Maybe when you are done making the note, we can talk about the possibility of cloning your notes with  a change of name and what it will cost to do such. Thanks

**Reply by JenuelDev** [reply]
is this a website? or software?
